tests/functional/generate_pages.py

```diff
@@ -651,6 +697,10 @@ def process_block(
             append_spec_page_import(block_context, spec_file)
 
 
+def _get_definitions_in_block_contents(block_contents):
+    return [element for element in block_contents if element.get("definition")]
```

Comment:
Given we don't need the return value from the lement, it would be clearer as `definition in element`
```suggestion
    return [element for element in block_contents if "definition" in element]
```

Reply:
Meh